views/home.erb

```diff
@@ -5,6 +5,15 @@
 <h1 class="dcf-txt-h3">Welcome to Innovation Studio Manager</h1>
 
 <h5>Hello, <%= @user.full_name %> (<%= @user.username %>)</h5>
+<% if @user.expiration_date > Date.today%>
+    <% if @user.expiration_date > (Time.now + (2*7*24*60*60)) %>
+        <h2 > Your Membership will expire on: <%= @user.expiration_date.to_formatted_s(:long_ordinal)[0...-5] %></h2>
```

Comment:
To avoid a DRY violation here with having ```Your Membership will expire on: <%= @user.expiration_date.to_formatted_s(:long_ordinal)[0...-5]``` in here twice you can do something like this pseudo-ish code below with a ternary operator in the inline style attribute.
```
if user.expiration_date > today:
    <h2 style="<%= user.expiration_date > now + 14 days ? "" : "color:yellow;" %>">Your Membership will 
     expire on: <%= @user.expiration_date.to_formatted_s(:long_ordinal)[0...-5]</h2>
else:
    <h2 style="color:red;">Your membership is inactive</h2>
```

Reply:
Done. Good suggestion!